contrib/cirrus/build_release.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+source $(dirname $0)/lib.sh
+
+req_env_var TEST_REMOTE_CLIENT OS_RELEASE_ID GOSRC
+
+cd $GOSRC
+
+if [[ "$TEST_REMOTE_CLIENT" == "true" ]] && [[ -z "$CROSS_PLATFORM" ]]
+then
+    CROSS_PLATFORM=linux
+fi
+
+if [[ -n "$CROSS_PLATFORM" ]]
+then
+    echo "Compiling podman-remote release archive for ${CROSS_PLATFORM}"
+    case "$CROSS_PLATFORM" in
+        linux) ;&
+        windows) ;&
+        darwin)
+            make podman-remote-${CROSS_PLATFORM}-release
+            ;;
```

Comment:
@jwhonce @ashley-cui I think the windows/darwin sections above are probably a good place to call into whatever is needed to produce msi and flatpack out of the release zip files `make` is going to emit.

Conveniently at this point in automation, a fully functional local podman is guaranteed.  So if your tooling can live in a container-image, it could easily be used here to do the business.  Come bother me when you're ready and I can assist with the integration.

Reply:
@cevich I'm using

libmsi1-0.99-2.fc30.x86_64
msitools-0.99-2.fc30.x86_64

I don't know if those are in your packaging container yet.  But the next inclusion if you could add those.